pkg/kubelet/config/flags.go

```diff
@@ -51,8 +49,6 @@ type ContainerRuntimeOptions struct {
 // AddFlags adds flags to the container runtime, according to ContainerRuntimeOptions.
 func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	// General settings.
-	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible value: 'remote'.")
-	fs.MarkDeprecated("container-runtime", "will be removed in 1.27 as the only valid value is 'remote'")
```

Comment:
can you please update the description of `pod-infra-container-image` in this same PR. So there will be no follow up PRs attempting to remove it